packages/desktop-electron/package.json

```diff
@@ -10,7 +10,6 @@
   "main": "index.js",
   "build": {
     "appId": "com.shiftreset.actual",
-    "afterSign": "./afterSignHook.js",
```

Comment:
Can you instead rename this so it stays around? In the future we should figure out how to code sign so I'd like to remember what we need to do here. Something like:

```
  "// afterSign": "./afterSignHook.js"
```

Reply:
Yeah, that works. I was trying to get this working with a CLI config, but the code ignores it for this.

Reply:
Heads up that `electron-builder` has some validation of this config so commenting them out like this results in errors.